lib/salus/scanners/gosec.rb

```diff
@@ -54,6 +54,12 @@ def run
       end
     end
 
+    def version
+      shell_return = run_shell('gosec --version')
+      # stdout looks like "Version: 2.4.0\nGit tag: v2.4.0\nBuild date: 2020-07-24T07:54:54Z"
+      shell_return.stdout.split('Version:')[1].split("\n")[0].strip
```

Comment:
Food for thought, we *could* access these in a safer fashion:

`shell_return.stdout&.split('Version:')&.dig(1)&.split("\n")&.dig(0)&.strip
`
And then we'd eliminate rescuing from StandError when we send to bugsnag.

Reply:
modified accordingly